ppa/archive/tests/test_eebo_tcp.py

```diff
@@ -39,32 +39,53 @@ def test_eebo_tcp_page_contents():
     )
 
 
+# ruff: noqa: E501
```

Comment:
Given that the line-too-long check has been turned off, why was the `PAGE_WITH_MULTIPLE_NOTES` broken into so many lines?